targets/TARGET_STM/lp_ticker.c

```diff
@@ -151,7 +151,7 @@ void lp_ticker_init(void)
 
     NVIC_SetVector(LPTIM1_IRQn, (uint32_t)LPTIM1_IRQHandler);
 
-#if !(TARGET_STM32L4)
+#if !(TARGET_STM32L4) && !(TARGET_STM32H7)
```

Comment:
This should be rebased since #8960 merge :-(

Reply:
OK. Rebased done